admin/src/translations/es.json

```diff
@@ -0,0 +1,48 @@
+{
+    "plugin.name": "Calendario",
+    "plugin.tagline": "Visualice sus eventos",
+    "view.calendar.state.empty.configure-settings.message": "Configure los ajustes antes de acceder al calendario",
+    "view.calendar.state.empty.configure-settings.action": "Ajustes",
+    "view.calendar.action.create-entry": "Crear nueva {colección}",
+    "view.calendar.action.select-date": "Seleccionar fecha",
+    "view.calendar.action.today": "Hoy",
+    "view.calendar.action.month": "Mes",
+    "view.calendar.action.week": "Semana",
+    "view.calendar.action.day": "Día",
+    "view.calendar.locale": "es-ES",
+    "view.calendar.first-day-of-week": "0",
+    "view.settings.title": "Ajustes del calendario",
+    "view.settings.subtitle": "Configure el complemento como mejor se adapte a sus necesidades",
+    "view.settings.action.save": "Guardar",
+    "view.settings.section.general.title": "Configuración general",
+    "view.settings.section.general.collection.label": "Elija su colección",
+    "view.settings.section.general.title.label": "Elija su campo de título",
+    "view.settings.section.general.title.none": "Sin campo de título",
+    "view.settings.section.general.default-duration.label": "Elija la duración predeterminada de su evento",
+    "view.settings.section.general.default-duration.30min": "30 minutos",
+    "view.settings.section.general.default-duration.1h": "1 hora",
+    "view.settings.section.general.default-duration.1.5h": "1,5 horas",
+    "view.settings.section.general.default-duration.2h": "2 horas",
+    "view.settings.section.general.start.label": "Elija su campo de inicio",
```

Comment:
I like how you used these on the hour option.
can we use
`Elija su campo de comienzo`
`Elija su campo de finalización`

Reply:
Absolutely! And thank you very much for adding the Hacktoberfest label 😊

Reply:
I´ve applied the "Elija su campo de comienzo", "Elija su campo de finalización" and "Sin campo de finalización". I hope it´s good to go 😊